WpNginxConfig.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen 443 ssl;
+    listen [::]:443 ssl;
```

Comment:
why do we need this line?

Reply:
listen 443 ssl = this line makes nginx listen on 443 port (ipv4 addresses connect on ssl https protocol)
listen [::]:443 ssl = same but with ipv6 addresses
Secure way of browsing a page, both ways communication is encrypted